src/components/NewMovie/NewMovie.tsx

```diff
@@ -13,33 +37,40 @@ export const NewMovie = () => {
       <TextField
         name="title"
         label="Title"
-        value=""
-        onChange={() => {}}
+        value={title}
+        onChange={(event) => setTitile(event)}
```

Comment:
```suggestion
        onChange={setTitile}
```
There is no event here. TextField component calls your handler with field value, so you don't need wrapper here.

Fix it everywhere.